examples/PPO/agent.py

```diff
@@ -0,0 +1,113 @@
+#   Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import parl
+import paddle
+import numpy as np
+from parl.utils.scheduler import LinearDecayScheduler
+
+
+class PPOAgent(parl.Agent):
+    """ Agent of PPO env
+
+    Args:
+        algorithm (`parl.Algorithm`): algorithm to be used in this agent.
+        config (dict): configs that used in this agent
+    """
+
+    def __init__(self, algorithm, config):
+        super(PPOAgent, self).__init__(algorithm)
+
+        self.config = config
+        if self.config['lr_decay']:
+            self.lr_scheduler = LinearDecayScheduler(
+                self.config['initial_lr'], self.config['num_updates'])
+
+    def predict(self, obs):
+        """ Predict action from current policy given observation
+
+        Args:
+            obs (np.array): observation
+        """
+        obs = paddle.to_tensor(obs, dtype='float32').unsqueeze(0)
+        action = self.alg.predict(obs)
+        action_numpy = action.detach().numpy()[0]
+        return action_numpy
+
+    def sample(self, obs):
+        """ Sample action from current policy given observation
+
+        Args:
+            obs (np.array): observation
+        """
+        obs = paddle.to_tensor(obs, dtype='float32')
+        value, action, action_log_probs, action_entropy = self.alg.sample(obs)
+        return value.detach(
+        ).numpy(), action.detach().numpy()[0], action_log_probs.detach().numpy(
+        )[0], action_entropy.detach().numpy()
+
+    def value(self, obs):
```

Comment:
please add doc for value, describe the shape of the input in this file

Reply:
done